packages/expo-cli/src/appleApi/authenticate.ts

```diff
@@ -190,3 +230,68 @@ function _formatTeam({ teamId, name, type }: FastlaneTeam): Team {
     inHouse: type.toLowerCase() === APPLE_IN_HOUSE_TEAM_TYPE,
   };
 }
+
+async function getLastUsedAppleIdAsync(): Promise<string | undefined> {
```

Comment:
dunno what the convention in cli is, but in `www`, if there is an absence of value, we prefer to return `null` instead
```suggestion
async function getLastUsedAppleIdAsync(): Promise<string | null> {
```

Reply:
this is a the type required by the prompt `string | undefined`